include/os/freebsd/spl/sys/condvar.h

```diff
@@ -143,7 +143,13 @@ cv_timedwait_sig(kcondvar_t *cvp, kmutex_t *mp, clock_t timo)
 }
 
 #define	cv_timedwait_io cv_timedwait
+#define	cv_timedwait_idle cv_timedwait
 #define	cv_timedwait_sig_io cv_timedwait_sig
+#define	cv_wait_io	cv_wait
+#define	cv_wait_io_sig	cv_wait_sig
+#define	cv_wait_idle	cv_wait
+#define	cv_timedwait_io_hires cv_timedwait_hires
+#define	cv_timedwait_idle_hires cv_timedwait_hires
```

Comment:
nit: while you're here it would be nice to indent these all to the same level for readability.